Add integration test (#46)
aiven/aiven-kafka-connect-s3: README.md

timestamp.source=wallclock
```

## Development

### Integration testing

Integration tests are implemented using JUnit, Gradle and Docker.

To run them, you need:
- Docker installed.

> Integration testing doesn't require a valid AWS credentials.

Reviewer: ```suggestion
Integration testing doesn't require valid AWS credentials.
```

Author: - [ ] Fix grammar

> To simulate AWS S3 behaviour tests uses [LocalStack](https://github.com/localstack/localstack-java-utils) provided by AWS.

Reviewer: Localstack is not really provided by AWS
```suggestion
To simulate AWS S3 behaviour tests uses [LocalStack](https://github.com/localstack/localstack-java-utils).
```

Author: Indeed. Not yet ;)))
- [ ] Remove comment

> To simulate AWS S3 behaviour tests uses [LocalStack](https://github.com/localstack/localstack-java-utils) provided by AWS.

Reviewer: ```suggestion
To simulate AWS S3 behaviour the tests use [LocalStack](https://github.com/localstack/localstack-java-utils) provided by AWS.
```

Author: How about without `the`, but with comma?
- [ ] fix grammar
